meerk40t/svgelements.py

```diff
@@ -190,22 +201,56 @@
     + "|"
     + SVG_TRANSFORM_SKEW_Y
 )
-PATTERN_TRANSFORM_UNITS = (
-    PATTERN_LENGTH_UNITS + "|" + PATTERN_ANGLE_UNITS + "|" + PATTERN_PERCENT
+PATTERN_TRANSFORM_UNITS = r"|".join(
+    (PATTERN_LENGTH_UNITS, PATTERN_ANGLE_UNITS, PATTERN_PERCENT)
 )
+TRANSFORM_UNIT_SINGULARS = {
+    "": "",
+    "%": "%",
+    "cm": "cm",
+    "cms": "cm",
+    "mm": "mm",
+    "mms": "mm",
+    "in": "in",
+    "ins": "in",
+    "inch": "in",
+    "inches": "in",
+    "mil": "mil",
+    "mils": "mil",
```

Comment:
Mil is not a unit within the CSS length routine. If you wish additional parsing be done here it should be done within the meerk40t project and not the svgelements project.

Reply:
As an aside, MK uses "turns" instead of "turn" and "mils" instead of "mil". I will address these.